src/MidnightExample/Ledger.lagda

```diff
@@ -214,7 +215,7 @@ non-zero, the proofs are identical.
 private variable s s' : LedgerState
                  b : Block
 
-open import Algebra.Properties.AbelianGroup +-0-abelianGroup
+open import Algebra.Properties.AbelianGroup +-0-abelianGroup using (identityʳ-unique) 
```

Comment:
It's been a while since I wrote this, is there a reason why this import isn't at the top? If not, I think that'd be a good addition here